.github/workflows/docker.yml

```
@@ -25,18 +41,26 @@ jobs:
       matrix:
         os: [Linux, macOS]
         arch: [default, complex]
-        include:
+        platform: [linux/amd64, linux/arm64]
+        # Cannot use inputs in exclude clauses, so add a dummy matrix dim
+        is_dev_build:
```

use same name as `inputs`
```suggestion
        build_dev:
```
maybe on a single line

---

Actionlint says that putting it on a single line is not acceptable.